docs/faq.md

```diff
@@ -21,7 +21,7 @@ the documentation sources.
 
 ### How much does Engine cost?
 
-Docker Engine is 100% free. It is open source, so you can use it without paying.
+The open source Docker Engine is available for free. Commercial support is also available for the Docker Engine as a standalone product or as part of the Docker Datacenter subscription. Pricing details are available at [https://www.docker.com/pricing](www.docker.com/pricing)
```

Comment:
There is no header at the top of that page that explicitly states it is open source docs. As this is just mixed in with all of the Docker documentation, I would ask we either state at the top these are ONLY open source FAQ's or they are Docker FAQ's and that should include both.